remote/inc/patterns/image-with-text.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Image with text
+ */
+return array(
+	'title'      => __( 'Image with text', 'remote' ),
+	'categories' => array( 'text', 'columns' ),
+	'content'    => '<!-- wp:image {"align":"wide","sizeSlug":"large","linkDestination":"none","className":"is-style-skatepark-aside-caption"} -->
```

Comment:
Is the skatepark classname needed here?

Reply:
ugh, how copy paste betrays me, how did I manage that? I'll fix it in a follow-up, thanks for catching that!